modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -1255,6 +1321,22 @@ public synchronized void rescheduleScaleDown(long delay, Runnable runnable, int
             scaleDownTaskDelay = delay;
         }
 
+        /**
+         * Reschedules existing partition distribution reset task, if it is not started yet and the delay of this task is not immediate,
+         * or schedules new one, if the current task cannot be canceled.
+         *
+         * @param delay Delay to start runnable in seconds.
+         * @param runnable Custom logic to run.
+         * @param zoneId Unique id of a zone to determine the executor of the task.
+         */
+        public synchronized void reschedulePartitionDistributionReset(long delay, Runnable runnable, int zoneId) {
+            stopPartitionDistributionReset();
+
+            partitionDistributionResetTask = executor.schedule(runnable, delay, SECONDS, zoneId);
```

Comment:
It worth adding Seconds postfix wherever needed to partitionDistributionResetTimeout/Delay. E.g. in DistributionZonesHighAvailabilityConfiguration

Reply:
done